game_eggs/steamcmd_servers/neosvr/egg-neos-v-r.json

```diff
@@ -0,0 +1,82 @@
+            "container": "ghcr.io\/pterodactyl\/installers:debian",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "Steam Account User",
+            "description": "The username for your Steam account.",
+            "env_variable": "STEAM_USER",
```

Comment:
is a steam user realy needed? normaly if you have beta code it can be downloaded  with anonymous

Reply:
Without logging in, you get this error from steamcmd: `ERROR! Password check for AppId 740250 returned error Access Denied.`

Reply:
oké fine then, but please add a meantion to the README for this that that it has to be a steam account that owns the game

Reply:
I have updated the readme in commit f8ce51ab5b6ae8fe78a4b2c6fea3c0b89149d086 to include the following about Steam account requirements: `To use this egg you will need a beta code and a Steam account. Currently the beta code can only acquired by going to the Neos Patreon, subscribing to the "Gunter" level, and linking your Patreon to Discord. Then you will be able to see the #headless-client channel and the password will be pinned in the command after -betapassword. The account will also need to have Neos VR in its library. You can do this by running app_license_request 740250 in steamcmd if you have never installed Neos VR on this Steam account.`

Reply:
fine for me! thank you.